talisker/gunicorn.py

```diff
@@ -90,6 +89,9 @@ def get_extra(self, resp, req, environ, request_time):
         if request_id:
             extra['request_id'] = request_id
 
+        msg = "{} {}".format(extra['method'], extra['path'])
+        if extra['qs']:
+            msg += '?'
```

Comment:
I do think it's worth including, or else I wouldn't have included it ;P

It's what other tools do (e.g. squid) when logging similar urls. The purpose is to give a quick indicator to a reader that this url had a query string, signalling that they may need to look deeper. Part of the issue was that the querystrings can be very long, and they are already included in the structured data.